### ISSUE TITLE
«caı» & «rue»

### ISSUE THREAD

**Issue by ciuak** [toaq/roots#4]
"caı": {
      "official": {
        "english": "▯ is strong.",
        "gloss": "strong"
      },
      "frame": "c"
    },
    "rue": {
      "official": {
        "english": "▯ is weak/feeble.",
        "gloss": "weak"
      },
      "frame": "c"
    }

These seem quite vague. I gotta know what they're supposed to mean exactly.

Here's a few relevant definitions from Wiktionary:

> 1. Capable of producing great physical force.
>   > a big strong man; Jake was tall and strong
>   
> 2. Capable of withstanding great physical force.
>   > a strong foundation; good strong shoes
>   
> 3. (of water, wind, etc.) Having a lot of power.
>   > The man was nearly drowned after a strong undercurrent swept him out to sea.
>   
> 4. Determined; unyielding. 
>   > He is strong in the face of adversity.
>   
> 5. Highly stimulating to the senses.
>   > a strong light; a strong taste
>   
> 6. Having an offensive or intense odor or flavor.
>   > a strong smell
>   
> 7. Having a high concentration of an essential or active ingredient.
>   > a strong cup of coffee; a strong medicine

(<https://en.wiktionary.org/wiki/strong>, adjective senses 1–7 out of 16)

The official entries use senses 5 (laqcāı/laqrūe) and 4? (caıdūa); other entries from Toadūa seem to exhaust the other possible senses (1 – ‘ruekōı’; 2 – ‘caıjīo’; 3 – ‘ruedēa’). Some of these senses might overlap, but it's not as clear. The question, of course, is what Hoemāı intended the word to mean in the first place.

(In case we settle on something, we might need to add a new field to the file.)

– kủaq mỉ hủaqchī jí ka

**Comment by solpahi** [comment]
I think I prefer senses 1-3. **caıdūa** and, probably, **laqcāı** were always metaphorical compounds (which I made in a hurry because I had no other options shortly before publishing the language).

So **caı** should mean something like "strong, tough, capable of producing or withstanding force, unyielding" and **rue** "weak, feeble, incapable of producing or withstanding physical force, yielding". And then by extension it could be applied metaphorically to psychological phenomena.

**Comment by ciuak** [comment]
I agree with the above, but I'd prefer «caı» & «rue» just to entail (3). This would simplify things. The rest could then be easily derived:
- (1) «caıdēq»
- (2) «caıfēa»
- (4) «caımōe» (see?)
- (5) «caıgē»

(etc.)
«laqcāı» is alright, I think; «caıdūa» could be made into «duaıdūa» or simply «duajāq».
